packages/formation-react/src/components/AlertBox/AlertBox.jsx

```diff
@@ -146,6 +151,7 @@ AlertBox.propTypes = {
 /* eslint-enable consistent-return */
 
 AlertBox.defaultProps = {
+  scrollPosition: 'start',
```

Comment:
We may want to keep this as `'end'` for all the other AlertBoxes across the site that are currently expecting it to have that behavior, then in your code you can provide a custom `scrollPosition` prop with the value `'start'`

Reply:
Oops just read the description, `'start'` works as a default prop then assuming it doesn't break any of the tests/desired behavior! 🙂

Reply:
There is only one place where we are currently using the scroll behavior, and I made sure to pass in the correct prop in the FE PR to make sure to keep existing functionality.